rosie.py

```diff
@@ -18,15 +18,15 @@ def help():
 
 
 def run():
-    import rosie
+    from rosie.ceap import classifiers
```

Comment:
If we call it "CEAP", we would need the document and always explain what the abbreviation means. Up to this point, Rosie only works with Brazilian data, so would be safe to call it just "chamber_of_deputies".